M2/Macaulay2/m2/galois.m2

```diff
@@ -117,7 +117,9 @@ findGalois(ZZ,ZZ) := RingElement => opts -> (p,n) -> (
      else error (opts.Strategy | " is not a valid argument for' Strategy' option")
      )
 
-GF(ZZ,ZZ) := GaloisField => opts -> (p,n) -> (
+GF(ZZ,ZZ) := GaloisField => (
+    memo := new CacheTable;
+    opts -> (p,n) -> memo#(p, n, opts) ??= (
```

Comment:
Does `memoize` ignore options?

Reply:
Yeah, it doesn't work well here.  If I use `memoize(opts -> (p, n) -> ...`, then it uses the function `(p.n) -> ...` as the keys for the memo instead of the actual input.

```m2
i1 : scan({2, 3, 5, 7, 11}, GF)

i2 : peek memoizeValues lookup(GF, ZZ, ZZ)

o2 = MutableHashTable{OptionTable{PrimitiveElement => FindOne} => FunctionClosure[../src/macaulay2/M2/M2/Macaulay2/m2/galois.m2:120:44-129:6]}
                                  SizeLimit => 10000
                                  Strategy => null
                                  Variable => null

```